src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -263,7 +263,7 @@ public void trimToNewStartPosition( final int newStart ) {
             }
 
             // Record our new start boundary
-            cachedInterval = new SimpleInterval(cachedInterval.getContig(), newStart, cachedInterval.getEnd());
+            cachedInterval = new SimpleInterval(cachedInterval);
```

Comment:
It seems to have exposed a hole in the tests.  They pass just fine with my new bug.